ITR/data/base_providers.py

```diff
@@ -151,46 +183,39 @@ def _convert_benchmark_to_series(self, benchmark: IBenchmark, scope: EScope) ->
         :param scope: a scope
         :return: pd.Series
         """
-        s = pd.Series({p.year: p.value for p in benchmark.projections}, name=(benchmark.region, benchmark.sector, scope),
-                      dtype=f'pint[{benchmark.benchmark_metric.units}]')
+        s = pd.Series({p.year: p.value for p in benchmark.projections}, name=(benchmark.sector, benchmark.region, scope),
+                      dtype=f'pint[{str(benchmark.benchmark_metric)}]')
         return s
 
-    def _get_projected_intensities(self, scope: EScope = EScope.S1S2) -> pd.DataFrame:
```

Comment:
as we remove this method - whioch method calculates intensities now?

Reply:
Yes.  _get_intensity_benchmarks now does this.  It could be that it's better to rename this function to `_get_projected_intensities` but keep everything else the same.  I haven't thought too much about this name game.

The history of the change is that we may have companies with named regions not supported by benchmarks, in which case we need to defer to the Global benchmark values.  In the course of implementing this deferral process, I strongly preferred the implementation of finding *all* missing benchmarks against the dataframe, and using Pandas patterns to substitute "Global" for the missing regions, rather than calling this function to do that logic row-by-row each time.

Generally, it's not very Pandas-like to iterate by rows when the problem can be expressed in terms of DataFrame-level actions.  So I eliminated it to avoid confusion.